lib/rust/parser/generate-java/java/org/enso/syntax2/Parser.java

```diff
@@ -26,7 +26,7 @@ public final class Parser implements AutoCloseable {
       System.load(parser.getAbsolutePath());
     } catch (URISyntaxException | LinkageError e) {
       File root = new File(".").getAbsoluteFile();
-      if (!searchFromDirToTop(e, root, "target", "rust", "debug", name)) {
+      if (!searchFromDirToTop(e, root, "target", "rust", "x86_64-apple-darwin", "debug", name)) {
```

Comment:
This is reverting [Marcin's Nov 9 change](https://github.com/enso-org/enso/commit/23e04f905f89b857ecd58a733f1aad4a63338513#diff-e237fc34931809c65628b8bcde58e32add8959b8e66940c46a5f63f557530302L30) done as part of #3859 - don't you have some old Rust tool chain?

Reply:
1.26 is the rustup version, which I thought was the question, and the latest version. 
```
rustup 1.26.0 (5af9b9484 2023-04-05)
info: This is the version for the rustup toolchain manager, not the rustc compiler.
info: The currently active `rustc` version is `rustc 1.68.0-nightly (1e4f90061 2023-01-11)`
```

Reply:
Rustup is not Rust toolchain, it is Rust toolchain installer :) 

This is a little worrying that a revert of Marcins commit is needed here. @JaroslavTulach would you be able to ask Marcin about it, please? Maybe he would remember why he did it.

Reply:
Sorry for the confusion. I've undone that change locally and I'm trying to run clean / build and it seems to work, so maybe even though at some stage I thought I needed that extra folder in the path in fact it's not needed. I'm still not too sure how the full build chain works...

Reply:
Ok, perfect. Thanks so much for the update! And thanks @JaroslavTulach for catching it!